main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
This would not loop 10 times as required in the prompt. Think through why that is (hint: double check your loop condition, sentinel value, and the starting value!)

Another thing to think through: are while loops the best to use for this prompt? Why or why not?
<details>
  <summary>Spoiler</summary>
  Since we know how many times this needs to increment and that it will increment each time, a for loop would be more readable. Lines 3-4 can and 22-23 can be condensed to:

```suggestion
for _ in range(10):
```
This would start the iterator at 0 and loop through until it hits 10 (not including 10), which means it iterates 10 times as required.
</details>